nipy/algorithms/clustering/tests/test_gmm.py

```diff
@@ -110,7 +111,7 @@ def test_em_selection():
     dim = 2
     x = np.concatenate((nr.randn(100, dim), 3 + 2 * nr.randn(100, dim)))
 
-    krange = range(1, 10)
+    krange = list(range(1, 10))
```

Comment:
Also unnecessary list construction. (`krange` iterated over in [best_fitting_GMM](https://github.com/matthew-brett/nipy/blob/more-py3-fixes/nipy/algorithms/clustering/gmm.py#L134).)